trojsten/submit/models.py

```diff
@@ -92,6 +94,7 @@ class Submit(models.Model):
     )
     points = models.DecimalField(max_digits=5, decimal_places=2, verbose_name="body")
 
+    text = models.CharField(max_length=512, verbose_name="text", blank=True)
```

Comment:
V súbore trojsten/contests/models.py je
text_submit_solution = models.CharField(max_length=128,...
Tu je 512. Nechce to byť rovnaké?

Reply:
V skutočnosti to môžeme zmenšiť na 32, nie? Dlhšie riešenie asi nebudeme mať. Neviem, aký to má efekt, či na tom záleží alebo nie.

Reply:
Asi na tom veľmi nezáleží

Reply:
kazdopadne to dam na rovnaku hodnotu, nech v tom nie je bordel

Reply:
Dal som to na 512 v oboch